app/assets/javascripts/application.js

```diff
@@ -173,7 +173,8 @@ var shortcuts = {
   79:  openCurrentLink,   // o
   191: openModal,         // ?
   190: sync,              // .
-  82:  sync               // r
+  82:  sync,              // r
+  27:  clearFilters
```

Comment:
I guess you need a '// esc' like the above lines

Reply:
Oh, missed that.

Reply:
Fixed.